extensions/resource-deployment/src/ui/toolsAndEulaSettingsPage.ts

```diff
@@ -0,0 +1,519 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { EOL } from 'os';
+import * as nls from 'vscode-nls';
+import { AgreementInfo, DeploymentProvider, ITool, ResourceType, ResourceTypeOptionValue, ToolStatus } from '../interfaces';
+import { createFlexContainer } from './modelViewUtils';
+import * as loc from '../localizedConstants';
+import { IToolsService } from '../services/toolsService';
+import { getErrorMessage } from '../common/utils';
+import { ResourceTypePage } from './resourceTypePage';
+import { ResourceTypeWizard } from './resourceTypeWizard';
+
+const localize = nls.loadMessageBundle();
+
+/**
+ * This page contains license agreements, deployment options and Tools table for deployment types. Should be included in all custom deployment types that opens a wizard for deployment.
+ * Earlier this was a part of the resourceTypePicker dialog
```

Comment:
I wouldn't bother including historical comments like this - it's generally not going to mean anything to people looking over this later.